packages/babel-parser/test/fixtures/typescript/module-namespace/head-declare/output.json

```diff
@@ -80,6 +80,99 @@
+          "name": "declare"
+        }
+      },
+      {
+        "type": "ExpressionStatement",
+        "start":94,"end":100,"loc":{"start":{"line":7,"column":0},"end":{"line":7,"column":6}},
+        "expression": {
+          "type": "Identifier",
+          "start":94,"end":100,"loc":{"start":{"line":7,"column":0},"end":{"line":7,"column":6},"identifierName":"module"},
+          "name": "module"
+        }
+      },
+      {
+        "type": "ExpressionStatement",
+        "start":101,"end":106,"loc":{"start":{"line":8,"column":0},"end":{"line":8,"column":5}},
+        "expression": {
+          "type": "StringLiteral",
+          "start":101,"end":106,"loc":{"start":{"line":8,"column":0},"end":{"line":8,"column":5}},
+          "extra": {
+            "rawValue": "foo",
+            "raw": "'foo'"
+          },
+          "value": "foo"
+        }
+      },
+      {
+        "type": "BlockStatement",
+        "start":107,"end":109,"loc":{"start":{"line":9,"column":0},"end":{"line":9,"column":2}},
+        "body": [],
+        "directives": []
+      },
+      {
+        "type": "ExpressionStatement",
+        "start":111,"end":117,"loc":{"start":{"line":11,"column":0},"end":{"line":11,"column":6}},
+        "expression": {
+          "type": "Identifier",
+          "start":111,"end":117,"loc":{"start":{"line":11,"column":0},"end":{"line":11,"column":6},"identifierName":"module"},
+          "name": "module"
+        }
+      },
+      {
+        "type": "ExpressionStatement",
+        "start":118,"end":123,"loc":{"start":{"line":12,"column":0},"end":{"line":12,"column":5}},
+        "expression": {
+          "type": "StringLiteral",
+          "start":118,"end":123,"loc":{"start":{"line":12,"column":0},"end":{"line":12,"column":5}},
+          "extra": {
+            "rawValue": "bar",
+            "raw": "'bar'"
+          },
+          "value": "bar"
+        }
+      },
+      {
+        "type": "BlockStatement",
+        "start":124,"end":126,"loc":{"start":{"line":13,"column":0},"end":{"line":13,"column":2}},
+        "body": [],
+        "directives": []
+      },
+      {
+        "type": "ExpressionStatement",
+        "start":128,"end":142,"loc":{"start":{"line":15,"column":0},"end":{"line":15,"column":14}},
+        "expression": {
+          "type": "Identifier",
+          "start":128,"end":135,"loc":{"start":{"line":15,"column":0},"end":{"line":15,"column":7},"identifierName":"declare"},
+          "name": "declare"
+        }
+      },
+      {
+        "type": "ExpressionStatement",
+        "start":143,"end":148,"loc":{"start":{"line":16,"column":0},"end":{"line":16,"column":5}},
+        "expression": {
+          "type": "StringLiteral",
+          "start":143,"end":148,"loc":{"start":{"line":16,"column":0},"end":{"line":16,"column":5}},
+          "extra": {
+            "rawValue": "baz",
+            "raw": "'baz'"
+          },
+          "value": "baz"
+        }
+      },
```

Comment:
I just discovered this is missing an `ExpressionStatement` for the `module` keyword. It is the same for `namespace` declarations.